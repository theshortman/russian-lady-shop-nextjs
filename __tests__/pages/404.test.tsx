import { render } from "@testing-library/react";
import Page404 from "@/pages/404";

test("renders page 404 unchanged", () => {
  const { container } = render(<Page404 />);
  expect(container).toMatchSnapshot();
});
