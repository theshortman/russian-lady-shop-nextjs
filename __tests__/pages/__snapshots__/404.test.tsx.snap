// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders page 404 unchanged 1`] = `
<div>
  <div>
    <h1>
      Not Found (#404)
    </h1>
    <p
      class="alert alert-danger"
    >
      Страница не найдена.
    </p>
  </div>
</div>
`;
